src/hotspot/share/opto/loopnode.cpp

```diff
@@ -4256,7 +4271,6 @@ void PhaseIdealLoop::build_loop_late_post( Node *n ) {
     case Op_LoadS:
     case Op_LoadP:
     case Op_LoadBarrierSlowReg:
-    case Op_LoadBarrierWeakSlowReg:
```

Comment:
comment that Op_LoadBarrierWeakSlowReg is introduced by ZGC and can be removed according to the original change